plugin/utils/gapic_utils.py

```diff
@@ -37,17 +37,39 @@ def read_from_gapic_yaml(yaml_file):
         gapic_yaml = yaml.load(f)
 
     collections = {}
+    fixed_collections = {}
+
+    all_collections = []
```

Comment:
Why is all_collections required - couldn't you just have all_entities here, and call extend instead of append?

Reply:
@andreamlin any comment on this? Seems like we could just have:
```python
all_entities = []
all_entities.extend(gapic_yaml['collections'])
for interface in gapic_yaml.get('interfaces', []):
  if 'collections' in interface:
    all_entities.extend(interface['collections'])
```

Reply:
oh yeah that's better